lib/Backend/Lower.cpp

```diff
@@ -3517,6 +3517,7 @@ Lowerer::LowerNewScObjectLiteral(IR::Instr *newObjInstr)
 
     //#if 0 TODO: OOP JIT, obj literal types
     // should pass in isShared bit through RPC, enable for in-proc jit to see perf impact
+#pragma prefast(suppress: __WARNING_LOGICALORNONZERO, "silence unnecessary error in release build, where CONFIG_FLAG(OOPJITMissingOpts) is #defined to be 0")
     Js::DynamicType * literalType = func->IsOOPJIT() || !CONFIG_FLAG(OOPJITMissingOpts) ? nullptr : *(Js::DynamicType **)literalTypeRef;
```

Comment:
would be better to re-organize the code with something like if(CONFIG_FLAG(OOPJITMissingOpts)), instead of suppressing the warning? and can make it more readable.